commit ce0b5382185d
Author: Jonathan Kew <jkew@mozilla.com>
Date:   Tue Mar 20 18:02:18 2018 +0000

    Bug 1427480 - Increase the MAX_FONT_SIZE limit applied in cairo_ft_font to match the gfxFont size limit. r=lsalzman, a=RyanVM
    
    --HG--
    extra : source : 74ffac5bc156c537ff4b1cc3d98dbb2c61b3127d
    extra : histedit_source : ee241f50804b89b4d545aedc5503cf0cb7c767f7
---
 gfx/cairo/cairo/src/cairo-ft-font.c | 2 +-
 gfx/cairo/max-font-size.patch       | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git gfx/cairo/cairo/src/cairo-ft-font.c gfx/cairo/cairo/src/cairo-ft-font.c
index 9d0e4951a28e..56430740a20c 100644
--- gfx/cairo/cairo/src/cairo-ft-font.c
+++ gfx/cairo/cairo/src/cairo-ft-font.c
@@ -103,7 +103,7 @@ static setLcdFilterFunc setLcdFilter;
 #define MAX_OPEN_FACES 10
 /* This is the maximum font size we allow to be passed to FT_Set_Char_Size
  */
-#define MAX_FONT_SIZE 1000
+#define MAX_FONT_SIZE 2000
 
 extern FT_Face mozilla_NewFTFace(FT_Library aFTLibrary, const char* aFileName, int aFaceIndex);
 extern FT_Face mozilla_NewFTFaceFromData(FT_Library aFTLibrary, const uint8_t* aData, size_t aDataSize, int aFaceIndex);
diff --git gfx/cairo/max-font-size.patch gfx/cairo/max-font-size.patch
index 99be23906ab4..efa54cef646a 100644
--- gfx/cairo/max-font-size.patch
+++ gfx/cairo/max-font-size.patch
@@ -8,7 +8,7 @@ diff --git a/gfx/cairo/cairo/src/cairo-ft-font.c b/gfx/cairo/cairo/src/cairo-ft-
 +
 +/* This is the maximum font size we allow to be passed to FT_Set_Char_Size
 + */
-+#define MAX_FONT_SIZE 1000
++#define MAX_FONT_SIZE 2000
  
  /*
   * The simple 2x2 matrix is converted into separate scale and shape
